index.html

```diff
@@ -0,0 +1,130 @@
+          <article class="main_program_act">
+            <i class="fa fa-hand-peace"></i>
+            <h3 class="program_act">Metal</h3>
+            <p class="program_act_description">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vitae ratione earum accusamus excepturi dolores? Facere hic provident libero porro</p>
+          </article>
+          <article class="main_program_act">
+            <i class="fa fa-compact-disc"></i>
+            <h3 class="program_act">Rap</h3>
+            <p class="program_act_description">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vitae ratione earum accusamus excepturi dolores? Facere hic provident libero porro</p>
+          </article>
+          <article class="main_program_act">
+            <i class="fab fa-deezer"></i>
+            <h3 class="program_act">Fusion</h3>
+            <p class="program_act_description">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vitae ratione earum accusamus excepturi dolores? Facere hic provident libero porro</p>
+          </article>
+          <article class="main_program_act">
+            <i class="fa fa-microphone"></i>
+            <h3 class="program_act">Workshop</h3>
+            <p class="program_act_description">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vitae ratione earum accusamus excepturi dolores? Facere hic provident libero porro</p>
+          </article>
+        </div>
+        <button type="button" class="main_program_btn">Join The Festival</button>
+      </section>
+
+      <!-- participants section -->
+      <section id="participants" class="participants_section">
+        <h2 class="participants_header">Participants</h2>
+        <hr />
+        <div class="participants_group"></div>
+      </section>
+    </main>
+    <footer id="footer">
+      <!-- partners section -->
+      <section class="partners">
+        <h2 class="partners_header">Partners</h2>
+        <hr />
+        <div class="partners_logos">
+          <img class="partner_logo" src="img/partners/partner1.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner2.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner3.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner4.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner5.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner6.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner7.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner8.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner9.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner10.png" alt="partner_logo" />
+          <img class="partner_logo" src="img/partners/partner11.png" alt="partner_logo" />
```

Comment:
- [ ] **Required:** Consider the color scheme of this section. Some of the icons used here are not visible. Kindly comply with the color scheme used in the Template Design

Your Design | Behance Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/152004773-4fb667da-c622-409f-8a93-a819ab46acb2.png) | ![image](https://user-images.githubusercontent.com/27709832/152004605-2e822260-92ca-46c9-8be8-cde5a8290b51.png)